src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -137,6 +230,51 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ *   public class firstTest {
+ *     public firstTest() {
+ *     } // OK
+ *   }
+ *
+ *   public class secondTest {
+ *     public secondTest() {
+ *       return;
+ *     } // OK
```

Comment:
It should be violation.
Please double check your cli out put and comments

Reply:
I was in the same confusion. But writing one return statement like this is not showing violation. In previous closed PR, one guy had pointed out the same problem.

Reply:
@Binita-tech Can you clear up the confusion from the code and explain why there is no violation and how we can get one in this scenario?

Reply:
```
public class secondTest {
  public secondTest() {
    return; // OK
     }
```
`return` in this case means: we're done, go back to the point where the constructor was called.
In Java, we can only write return in constructor to stop execution but we can't write return with some value.
```
public class secondTest {
  public secondTest() {
    return 1; // violation
} 
```

Reply:
@Binita-tech Sorry if my comment confused you, but I was talking about connecting this with our check's code.

It will help us to show why no violation for this check shows up by pointing out the section in the code and providing an similar code example that would cause a violation with our check.

You say no violation shows from the check, @romani was under the impression there should be, so I am looking to show @romani where the disconnect is and help clarify things for him.

Reply:
```
@FileStatefulCheck
public final class ReturnCountCheck extends AbstractCheck {
```
Do I have to check the code below this to know why it is not causing a violation?
I am new to this and confused a little bit, please guide @rnveach.

Reply:
I create issue violation is expected but not placed until #11212